cluster_experiments/power_analysis.py

```diff
@@ -284,6 +284,8 @@ def _non_parallel_loop(
         n_detected_mde = 0
         for _ in tqdm(range(n_simulations), disable=not verbose):
             p_value = self._run_simulation((df, average_effect))
+            if verbose:
+                print(f"p_value of simulation run: {p_value}")
```

Comment:
perhaps we can format this:
```suggestion
                print(f"p_value of simulation run: {p_value:.3f}")
```

Reply:
done